### PR TITLE
[azure] Added forwarded tag for Azure logs

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Added forwarded tags for Azure logs.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by andrewkroh** [comment]
Hi @LaZyDK, can you add an entry to the changelog.yml file and bump the version in the manifest.yml to match.

**Comment by LaZyDK** [comment]
@andrewkroh Done :)

File: packages/azure/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.1.2"
+  changes:
+    - description: Added forwarded tags to all collected logs
```
Comment (andrewkroh) [inline packages/azure/changelog.yml:3]:
```suggestion
    - description: Added the `forwarded` tag by default to all log types.
```

**Review by andrewkroh — COMMENTED** [review]
LGTM. Let's see what CI testing says. There might be some expected test files that need updated to include new tag (e.g. `cd packages/azure && elastic-package test pipeline --generate`).

**Review by kaiyan-sheng — APPROVED** [review]
Hello sorry for the late response here! @LaZyDK Could you fix the conflict please? Thank you!!

**Comment by LaZyDK** [comment]
It has been done.

**Pull request merged**